starboard/tools/api_leak_detector/api_leak_detector.py

```diff
@@ -109,6 +109,8 @@
     'lseek',
     'malloc',
     'posix_memalign',
+    'PosixSocketWaiterAdd',
```

Comment:
why do you need these?

Reply:
They are renamed to SbPosixSocketWaiter* and moved to near other SbSocketWaiter functions in this file.